src/profiler/profiler.h

```diff
@@ -787,6 +787,10 @@ struct ProfileTask : public ProfileDuration {
     NVTX_ONLY_CODE(nvtx_duration_.reset(new nvtx::NVTXDuration(name)));
   }
 
+  void setDomain(ProfileDomain *domain) {
```

Comment:
Add `const` to ProfileDomain?

Reply:
We actually would not want that because domain_ is not constant